src/styles/main.scss

```diff
@@ -1,3 +1,62 @@
+@import "./variables.scss";
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 800px;
+
+  &__block {
+    box-sizing: border-box;
+    height: 100px;
+    width: 100px;
+    background-color: #eee;
+    border: 1px solid black;
+    padding: 10px;
+    transition: background-color 500ms, transform 500ms;
+
+    @for $cell from 1 through 31 {
+      &:nth-child(#{$cell}):before {
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        content: "#{$cell}";
+        display: flex;
+        justify-content: center;
+        line-height: 90px;
+      }
+    }
+  }
+
+  &__block:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+  }
```

Comment:
please check the checklist '[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.'
the hover should be inside the &__block